### PR TITLE
Use Bootstrap Utilities to space and justify content.

### PR DESCRIPTION
**Pull request by jwq1**

### CODE REVIEW

**Comment by jwq1** [comment]
screen shot 2018-06-24 at 9 05 18 am

Not perfect yet. It breaks at mobile screen sizes. The font size needs to come down and the text can't overlap.

**Comment by jwq1** [comment]
screen shot 2018-06-24 at 9 46 13 am


@hdavidzhu my html structure is the same for all three containers on the page. I don't have any extra styling in my stylesheet. However, the bottom container is misaligned. It is shifted to the left. Do you know why this could be? I cannot spot any difference between the html elements.

**Comment by jwq1** [comment]
The text is ignoring the padding for some reason. 
screen shot 2018-06-24 at 9 49 49 am

File: styles/styles.css [hunk 1]
```diff
@@ -17,6 +17,7 @@ header h1 {
   font-weight: 700;
   margin: 0;
   font-family: 'Helvetica';
+  justify-content: flex-start;
```
Comment (hdavidzhu) [inline styles/styles.css]: https://getbootstrap.com/docs/4.1/utilities/flex/#justify-content

File: styles/styles.css [hunk 2]
```diff
@@ -32,50 +33,10 @@ header p {
...
-  justify-content: left;
-  font-size: 36px;
-}
-
-.story-scene {
-  margin-bottom: 20px;
-  justify-content: left;
-  font-size: 36px;
+.storyline {
+  font-size: 2.5rem;
```
Comment (hdavidzhu) [inline styles/styles.css]: Text also follows the same sizing as columns:

https://getbootstrap.com/docs/4.1/content/typography/

https://getbootstrap.com/docs/4.1/utilities/text/

**Pull request merged**